packages/expo-maps/src/google/GoogleMapsModule.ts

```diff
@@ -0,0 +1,24 @@
+import { PermissionResponse, requireNativeModule } from 'expo-modules-core';
+import { Platform } from 'react-native';
+
+let NativeModule: any = null;
+
+if (Platform.OS === 'android') {
+  NativeModule = requireNativeModule('ExpoGoogleMaps');
+}
```

Comment:
```suggestion
const NativeModule = requireOptionalNativeModule('ExpoGoogleMaps')
```
Also, it would be nice to provide the module type and export it as default without re-exposing methods. See [`ImageModule.ts`](https://github.com/expo/expo/blob/main/packages/expo-image/src/ImageModule.ts)

Reply:
Fixed in https://github.com/expo/expo/pull/34551/commits/d7ae2a61f0b2bdd0c9e09dafef4f61bb6ecf4fab